### ISSUE TITLE
Arrows ('↑' and '↓') in en.xaml

### ISSUE THREAD

**Issue by VictoriousRaptor** [Flow-Launcher/Flow.Launcher.Plugin.Everything#55]
The current en.xaml file uses arrows ('↑' and '↓') to represent ascending and descending:

```
    <system:String x:Key="flowlauncher_plugin_everything_sort_by_ascending">↑</system:String>
    <system:String x:Key="flowlauncher_plugin_everything_sort_by_descending">↓</system:String>
```

Which has no corresponding translation in other languanges, so users in other languages see two 'Name', 'Path' or something else:

```
    <system:String x:Key="flowlauncher_plugin_everything_sort_by_ascending"></system:String>
    <system:String x:Key="flowlauncher_plugin_everything_sort_by_descending"></system:String>
```

![图片](https://user-images.githubusercontent.com/10308169/191548842-43bd4849-9c3d-4b17-a33c-6ff6bcc87817.png)

I guess it's because Crowdin can't deal with '↑' and '↓' correctly, there's no corresponding entry in l10n files.

We should consider adding it mannually for every language, or changing '↑' and '↓' to 'Ascending'and 'Descending'.

**Comment by jjw24** [comment]
Yep makes sense

**Comment by taooceros** [comment]
Let's see whether it is because the previous crowdin issue.

**Comment by taooceros** [comment]
What about make it unicode?

**Comment by jjw24** [comment]
PR merged, please update to v1.7.5 and let us know :)

**Comment by taooceros** [comment]
~~I think it cannot resolve it...Let's make it asc and desc maybe?~~

It works. I believes it is some trimming issue from crowdin before.
